src/blog/_posts/2017-12-19-opera-50.md

```diff
@@ -0,0 +1,196 @@
+
+
+## Device Memory API
+
+Using the new
+[Device Memory API](https://github.com/w3c/device-memory#the-web-exposed-api)
+developers can estimate the total amount of RAM on the device. This information
+could be used to deliver a suitably scaled version of the site to lower-end
+devices, or help put collected performance data into context to better
+understand the behavior of a web application.
+
+
+## Other web platform features in this release
+
+### Blink > CSS
+
+* Developers can now
+  [make pixel-level adjustments](http://w3c-test.org/html/semantics/embedded-content/the-img-element/sizes/parse-a-sizes-attribute.html)
+  using the new
+  [Q length unit](http://thenewcode.com/1158/The-Q-Continuum-Using-q-Measurements-In-CSS),
+  which is especially useful on small viewports.
+
+### Blink > Fonts
+
+* [font-variant-east-asian](https://drafts.csswg.org/css-fonts-3/#font-variant-east-asian-prop)
+  is now supported, allowing developers to control the
+  [usage of alternate glyphs](https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant-east-asian)
+  for East Asian languages like Japanese and Chinese.
+
+### Blink > HTML
+
+* To improve interoperability, Opera will fire `beforeprint` and `afterprint`
+  events as part of the printing standard, allowing developers to to annotate
+  the printed copy and edit the annotation after the printing command is done
+  executing.
+
+### Blink > JavaScript
+
+* Using
+  [`Promise.prototype.finally()`](https://developers.google.com/web/updates/2017/10/promise-finally),
+  a callback can now be registered to be invoked after a `Promise` has been
+  either fulfilled or rejected.
+* The
+  [`Intl.PluralRules` API](https://developers.google.com/web/updates/2017/10/intl-pluralrules)
+  allows developers to build applications that understand pluralization of a
+  given language by indicating which plural form applies for a given number and
+  language.
+
+### Blink > MediaStream
+
+* [`MediaStreamTrack.applyConstraints()`](https://developer.mozilla.org/en-US/docs/Web/API/MediaStreamTrack/applyConstraints)
+  is now supported for local video
+  [`MediaStreamTracks`](https://w3c.github.io/mediacapture-main/#mediastreamtrack),
+  including tracks obtained from
+  [`getUserMedia()`](https://w3c.github.io/mediacapture-main/#dom-mediadevices-getusermedia()),
+  [capture from media elements](https://w3c.github.io/mediacapture-fromelement/)
+  or screen capture.
+
+### Blink > Network
+
+* [Version 2](https://docs.google.com/document/d/18kQ_TFQns2HZ9KGs4bGDfiiOIRmOhI12oJBttvUlYLY/edit)
+  of
+  [NT LAN Manager (NTLM) API](https://msdn.microsoft.com/en-us/library/cc236621.aspx)
+  is now shipped, enabling applications to authenticate remote users and provide
+  session security when requested by the application.
+
+### Blink > Sensor
+
+* Thanks to contributors from engineers at Intel,
+  [an Origin Trial is now available](https://developers.google.com/web/updates/2017/09/sensors-for-the-web)
```

Comment:
We don't support origin trial AFAIK, so I'm not sure if we should be mentioning them? @bratell-at-opera ?

Reply:
No, I guess the origin trial bit is inaccurate. But the feature flag is there in opera://flags, so I guess you can still manually enable it in Opera.

But I suppose we might as well not mention a default-disabled optional feature.

Reply:
Yes, we don't have any of the infrastructure for origin trials so better to pretend they don't exist. Well, since they don't.